notifier.rst

```diff
@@ -90,7 +91,7 @@ Yunpian         ``symfony/yunpian-notifier``          ``yunpian://APIKEY@default
 
 .. versionadded:: 6.1
 
-    The 46elks and OrangeSms integrations were introduced in Symfony 6.1.
+    The KazInfoTeh, 46elks and OrangeSms integrations were introduced in Symfony 6.1.
```

Comment:
```suggestion
    The 46elk, OrangeSms and KazInfoTeh integrations were introduced in Symfony 6.1.
```

Please move new bridges to the end, thanks

Reply:
was replaced

Reply:
thanks